index.js

```diff
@@ -20,8 +20,8 @@ function apply(options, compiler) {
     {
       // Default Glob is any directory level of scss and/or sass file,
       // under webpack's context and specificity changed via globbing patterns
-      files: arrify(options.files || defaultFilesGlob).map((file) =>
-        path.join(context, '/', file)
+      files: arrify(options.files || defaultFilesGlob).map(
+        (file) => (path.isAbsolute(file) ? file : path.join(context, '/', file))
```

Comment:
I think this can be expressed as `path.resolve(context, file)` :thinking:.

```
path.join('foo', '/', 'bar') === path.join('foo', 'bar') // true; separator is extraneous
path.resolve('./foo', 'bar') // process.cwd() + '/bar'
path.resolve('/foo', 'bar') // /foo/bar
path.resolve('/foo', '/bar') // /bar
```